pkg/exporter/collectors.go

```diff
@@ -5,6 +5,7 @@ import "github.com/prometheus/client_golang/prometheus"
 var (
 	defaultLabels                = []string{"project", "topics", "kind", "ref", "variables"}
 	jobLabels                    = []string{"stage", "job_name", "runner_description"}
+	traceLabels                  = []string{"job_id", "trace_rule"}
```

Comment:
I believe we already have a metric for the `job_id` :thinking:

Reply:
Ah yes - probably I've just put it there to make the promql statement simpler for the new grafana dashboard:

`sort_desc(gitlab_ci_pipeline_job_trace_match_count{project=~"$PROJECT", ref=~"$REF", job=~"$JOB", trace_rule=~"$TRACE_RULE"} > 0)`

Reply:
It's a table view with ID in the first column. I've seen that you also have JobIDs in the Job Dashboard - by joining/merging the statements, right?